generators/app/templates/.eslintrc.ejs

```diff
@@ -3,5 +3,20 @@ module.exports = {
   plugins: ['flowtype']<% } %><% if (features.jest) { %>,
   env: {
     jest: true
-  }<% } %>
+  }<% } %><% if (features['babel-module-resolver']) { %>,
+  settings: { 
+    'import/resolver': { 
+      'babel-module': {  
+        alias: {
+          '~components': './src/app/components',
+          '~screens': './src/app/screens',
+          '~config': './src/config',
+          '~constants': './src/constants',
+          '~redux': './src/redux',
+          '~services': './src/services',
+          '~utils': './src/utils'
```

Comment:
Dunno, I don't like this character `~` because the eslint. Can we change ours wolox eslint regex config to avoid the empty line between each import?

Reply:
I've tried everything. If we use @, it thinks it's a dependency with [scope](https://docs.npmjs.com/misc/scope) and eslint's rule `import/no-extraneous-dependencies` shows an error. We shouldn't remove that rule, as it's useful to find imports of dependencies that aren't installed.

Here's an issue about this: https://github.com/benmosher/eslint-plugin-import/issues/807
And here's the open PR to fix this that doesn't seem to be going anywhere for the time being: https://github.com/benmosher/eslint-plugin-import/pull/914